src/coredump/coredump.c

```diff
@@ -286,16 +286,21 @@ static int make_filename(const char *context[_CONTEXT_MAX], char **ret) {
         if (!p)
                 return -ENOMEM;
 
+        s = filename_escape(context[CONTEXT_SIGNAL]);
+        if (!s)
+                return -ENOMEM;
```

Comment:
I agree, but adding signal number is going to increase the file name size by just 3 bytes.  Even though we have the crash reason inside the core file, but that would require us to unxz it, launch gdb to get the reason.   Instead if we have this information in the file name, then it would help us to get the reason quickly